frontend/src/scenes/experiments/ExperimentView/components.tsx

```diff
@@ -542,34 +516,55 @@ export function MakeDecisionModal({ experimentId }: { experimentId: Experiment['
+                        onClick={() => shipVariant({ selectedVariantKey, shouldStopExperiment })}
+                        type="primary"
+                    >
                         Ship variant
                     </LemonButton>
                 </div>
             }
         >
-            <div className="space-y-4">
+            <div className="space-y-6">
                 <div className="text-sm">
                     This action will roll out the selected variant to <b>100% of {aggregationTargetName}.</b>
                 </div>
-                <LemonSelect
-                    data-attr="metrics-selector"
-                    value={selectedVariantKey}
-                    onChange={(variantKey) => setSelectedVariantKey(variantKey)}
-                    options={sortedWinProbabilities.map(({ key }) => ({
-                        value: key,
-                        label: (
-                            <div className="space-x-2 inline-flex">
-                                <VariantTag experimentId={experimentId} variantKey={key} />
-                                {key === sortedWinProbabilities[0]?.key && (
-                                    <LemonTag type="success">
-                                        <b className="uppercase">Winning</b>
-                                    </LemonTag>
-                                )}
+                <div className="flex items-center">
+                    <div className="w-1/2 pr-4">
+                        <LemonSelect
+                            className="w-full"
+                            data-attr="metrics-selector"
+                            value={selectedVariantKey}
+                            onChange={(variantKey) => setSelectedVariantKey(variantKey)}
+                            options={sortedWinProbabilities.map(({ key }) => ({
+                                value: key,
+                                label: (
+                                    <div className="space-x-2 inline-flex">
+                                        <VariantTag experimentId={experimentId} variantKey={key} />
+                                        {key === sortedWinProbabilities[0]?.key && (
+                                            <LemonTag type="success">
+                                                <b className="uppercase">Winning</b>
+                                            </LemonTag>
+                                        )}
+                                    </div>
+                                ),
+                            }))}
+                        />
+                    </div>
+                    {!isExperimentStopped && (
+                        <>
+                            <LemonDivider className="my-0" vertical />
+                            <div className="w-2/5 pl-4">
+                                <LemonCheckbox
+                                    id="flag-enabled-checkbox"
+                                    label="Stop experiment"
```

Comment:
can we get a tooltip on this?  Something that, idk, explains why this checkbox exists at all, and the ramifications of stopping the experiment when shipping the variant vs keeping it going?  As a user, it's not immediately clear what checking/unchecking this box does within the modal context.  Tooltips are unreasonably effective and providing explanations without cluttering visuals, so IMO it feels like it'd be a nice lift to include.

Reply:
done! thanks for approving early. I also love tooltips btw :D